protocol/messenger_communities.go

```diff
@@ -1288,8 +1296,16 @@ func (m *Messenger) EditCommunityTokenPermission(request *requests.EditCommunity
 		return nil, err
 	}
 
-	response := &MessengerResponse{}
-	response.AddCommunity(community)
+	response, err := m.UpdateCommunityEncryption(community.ID())
```

Comment:
I think it would be faster if you passed the whole community here instead of just the ID. No need to hit the DB.